index.html

```diff
@@ -55,10 +57,234 @@ <h4>LET'S CONNECT</h4>
+                        Hello I’m a software developer! I can help you build a product,
+                        feature or website. Look through some of my work and experience! If
+                        you like what you see and have a project you need coded, don’t
+                        hestiate to contact me.
+                    </p>
+                    <h4 class="refer-link">LET'S CONNECT</h4>
+                    <ul class="icon-content-element">
+                        <li>
+                            <a href="https://github.com/Enoisong" target="_blank" rel="noopener noreferrer"><img
+                                    src="./images/github.svg" alt="github icon" /></a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/enobong-isong/" target="_blank"
+                                rel="noopener noreferrer"><img src="./images/Linkedin.svg" alt="LinkdIn icon" /></a>
+                        </li>
+                        <li>
+                            <a href="#" target="blank" rel="noopener noreferrer"><img src="./images/angellist.svg"
+                                    alt="Angellist icon" /></a>
+                        </li>
+                        <li>
+                            <a href="https://twitter.com/Enobongmisong" target="_blank" rel="noopener noreferrer"><img
+                                    src="./images/twitter.svg" alt="twitter icon" /></a>
+                        </li>
+                        <li>
+                            <a href="#" target="_blank" rel="noopener noreferrer"><img src="./images/medium.svg"
+                                    alt="Medium icon" /></a>
+                        </li>
+                    </ul>
+                    <button class="get-it-btn" type="submit">Get my resume</button>
+                </div>
+                <div id="languages-slider">
+                    <ul>
+                        <li id="languages-slider-1">
+                            Languages
+                            <img src="./images/Icons/arowB.png" alt="arrow bottom icon" />
+                        </li>
+                    </ul>
+                    <ul id="box-languages-items">
+                        <li class="box-languages">
+                            <img class="icon-1" src="./images/javaScript.png" alt="JavaScript icon" />
+                            <p class="lg-1">JavaScript</p>
+                        </li>
+                        <li class="box-languages">
+                            <img class="icon-1" src="./images/html5.png" alt="HTML5 icon" />
+                            <p class="lg-1">HTML</p>
+                        </li>
+                        <li class="box-languages">
+                            <img class="icon-1" src="./images/css3.png" alt="CSS3 icon" />
+                            <p class="lg-1">CSS</p>
+                        </li>
+                    </ul>
```

Comment:
Great job adding the box languages 👍. please, let there be space between these language so it can match the Figma design.

| Your Design | Figma Design |
| :---:   | :---: |
|<img width="351" alt="image" src="https://user-images.githubusercontent.com/73854999/219855166-007e325b-db81-474a-9a5a-b646d89c4b4f.png"> | <img width="351" alt="image" src="https://user-images.githubusercontent.com/73854999/219855154-bc232b2d-9fe0-425a-9298-0a40432786ae.png">|